client/css/style.css

```diff
@@ -1108,18 +1108,23 @@ background on hover (unless active) */
 }
 
 .scroll-down-arrow {
-	width: 48px;
-	height: 48px;
-	line-height: 48px;
+	width: 36px;
+	height: 36px;
+	line-height: 34px;
 	border-radius: 50%;
-	background: #fff;
-	color: #333;
-	border: 1px solid #84ce88;
+	background: var(--window-bg-color);
+	color: #84ce88;
```

Comment:
Do we want to make this color a css var?

Reply:
I can do that.

Reply:
I'm just thinking if we should make it a generic one (we use it a lot elsewhere it seems) or specifically for this button.

Reply:
I don't think a variable for a given color makes it any useful. Let me come up with something and you tell me if you like it or not.

Reply:
Done. Went a simple route, and if we ever need more granularity (different variables for border, text color, etc.), we can add them, but I think for now that's enough and provides some customization level already.